### PR TITLE
RHDEVDOCS-6254: Disabling shipwright CR

### PR DESCRIPTION
**Pull request by shivanisathe25**
Version(s): build-docs-1.1, build-docs-1.2

Issue: https://issues.redhat.com/browse/RHDEVDOCS-6254

Link to docs preview: https://85268--ocpdocs-pr.netlify.app/openshift-builds/latest/installing/installing-openshift-builds.html#ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc_installing-openshift-builds

QE review: @apoorvajagtap @avinal 
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by avinal — APPROVED** [review]
Looks good to me.

File: modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// This module is included in the following assembly:
+//
+// * installing/installing-openshift-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc_{context}"]
+= Disabling ShipwrightBuilds build controller and webhook pod
+
+You can disable `ShipwrightBuilds` CR without disabling the shared-resources CSI driver component. Disabling `ShipwrightBuilds` CR removes the `ShipwrightBuilds` build controller and `ShipwrightBuilds` build webhook pod from the `openshift-builds` namespace.
```
Comment (aspauldi) [inline modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc:9]: Can you include what the acronyms CR and CSI stand for? 

From the [Language](https://ccs-internal-documentation.pages.redhat.com/peer-review/#_language) section of the [Red Hat peer review guide](https://ccs-internal-documentation.pages.redhat.com/peer-review/):  "Acronyms are expanded on first use."

File: modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+.Procedure
+
+. Disable ShipwrightBuilds controller and webhook by running the following command:
```
Comment (aspauldi) [inline modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc:13]: Same comment/question as line 9.

File: modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
++
+[source,terminal]
+----
+oc patch openshiftbuild cluster --type=merge -p '{"spec":{"shipwright":{"build":{"state":"Disabled"}}}}'
+----
+
+The above command disables the build controller and webhook but does not affect the ShipwrightBuilds Custom Resource Definition (CRD).
```
Comment (aspauldi) [inline modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc:20]:
```suggestion
This command disables the build controller and webhook but does not affect the ShipwrightBuilds Custom Resource Definition (CRD).
```

**Review by aspauldi — COMMENTED** [review]
Hi, just a few comments!

**Comment by dfitzmau** [comment]
Hello @shivanisathe25 .

You'll need to [squash the commits](https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit?pli=1&tab=t.0#heading=h.epes4xf06pzk) from 2 into 1.

File: modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc [hunk 4]
```diff
@@ -0,0 +1,20 @@
+// This module is included in the following assembly:
+//
+// * installing/installing-openshift-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc_{context}"]
+= Disabling ShipwrightBuilds build controller and webhook pod
+
+You can disable the `ShipwrightBuilds` Custom Resource (CR) without disabling the shared-resources Container Storage Interface (CSI) driver component. Disabling `ShipwrightBuilds` CR removes the `ShipwrightBuilds` build controller and `ShipwrightBuilds` build webhook pod from the `openshift-builds` namespace.
```
Comment (dfitzmau) [inline modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc:9]:
```suggestion
You can disable the `ShipwrightBuilds` Custom Resource (CR) without disabling the shared-resources Container Storage Interface (CSI) driver component. Disabling the `ShipwrightBuilds` CR removes the `ShipwrightBuilds` build controller and `ShipwrightBuilds` build webhook pod from the `openshift-builds` namespace.
```

File: modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+.Procedure
+
+. Disable the ShipwrightBuilds controller and webhook by running the following command:
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc:13]: ShipwrightBuilds > `ShipwrightBuilds`

File: modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
++
+[source,terminal]
+----
+oc patch openshiftbuild cluster --type=merge -p '{"spec":{"shipwright":{"build":{"state":"Disabled"}}}}'
```
Comment (dfitzmau) [inline modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc:17]:
```suggestion
$ oc patch openshiftbuild cluster --type=merge -p '{"spec":{"shipwright":{"build":{"state":"Disabled"}}}}'
```

File: modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+----
+
+This command disables the build controller and webhook but does not affect the ShipwrightBuilds Custom Resource Definition (CRD).
```
Comment (dfitzmau) [inline modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc:20]: Add a plus (+) symbol to indent this paragraph with the procedure step.

Comment (dfitzmau) [reply modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc:20]: ShipwrightBuilds > `ShipwrightBuilds`

A CR or a CRD? `ShipwrightBuilds` CR is mentioned at the intro of the module but it's CRD in this part.

Comment (dfitzmau) [reply modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc:20]: AFAIK, A CRD is for creating custom resources types. A CR is for creating a resource based on the resource type defined in a CRD.

**Comment by dfitzmau** [comment]
Hi @shivanisathe25 . Good job! I've finished my review and added comments inline. Please contact me if you have any queries.

File: modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+:_mod-docs-content-type: PROCEDURE
+[id="ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc_{context}"]
+= Disabling ShipwrightBuilds build controller and webhook pod
+
+You can disable the `ShipwrightBuilds` Custom Resource (CR) without disabling the shared-resources Container Storage Interface (CSI) driver component and the `ShipwrightBuilds` Custom Resource Definition (CRD). Disabling the `ShipwrightBuilds` CR removes the `ShipwrightBuilds` build controller and `ShipwrightBuilds` build webhook pod from the `openshift-builds` namespace.
+
+.Procedure
+
+* Disable the ShipwrightBuilds controller and webhook by running the following command:
```
Comment (dfitzmau) [inline modules/ob-disabling-shipwrightBuilds-build-controller-and-webhook-pod.adoc:13]:
```suggestion
* Disable the `ShipwrightBuilds` controller and webhook by running the following command:
```

**Comment by dfitzmau** [comment]
build-docs-1.1

**Pull request merged**